virtdbg/vmx/ept.c

```diff
@@ -1,25 +1,15 @@
 #include <sync/lock.h>

```

Comment:
This whole file needs tons of cleanup, remove unneeded defines, comments, and so on. Make sure to use structs.